src/card.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>WITCH OF MOUNTAIN WINDS</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+    <link 
+      rel="shortcut icon" 
+      href="./images/svg/logo-colour.svg" 
+      type="image/png"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600&display=swap" rel="stylesheet">
+  </head>
+  <body class="page__body">
+    <header class="header">
+      <div class="container header__content">
+        <div class="header__top">
+          <div class="header__menu">
+            <a href="#menu" class="icon icon__menu header__nav--link"></a>
+          </div>
+          <div class="header__logo">
+            <a href="#">
+              <div class="icon icon__logo"></div>
+            </a>
+          </div>
+          <ul class="header__nav">
+            <li>
+              <a href="#shop" class="header__nav--link">
+                shop
+              </a>
+            </li>
+            <li>
+              <a href="#contacts" class="header__nav--link">
+                contact
+              </a>
+            </li>
+            <li>
+              <a href="#about" class="header__nav--link">
+                about
+              </a>
+            </li>
+          </ul>
+          <div class="header__basket">
+            <a href="#basket" class="icon icon__basket"></a>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <nav class="page__menu menu" id="menu">
```

Comment:
BEM class naming is not used properly. Use double underscores for elements and double hyphens for modifiers, e.g. 'block__element--modifier'.